test/run_tests.pl

```diff
@@ -20,6 +20,11 @@ BEGIN
         if $ENV{VERBOSE_FAILURES_PROGRESS} || $ENV{VFP};
 }
 
+# Support running jobs in parallel
+BEGIN {
+    $ENV{HARNESS_JOBS} = "4" if (!defined $ENV{HARNESS_JOBS});
```

Comment:
This looks like 4 parallel jobs still becomes the default here, which @levitte did not recommend.

BTW, I would actually prefer having this as the default.

Reply:
If we keep it, the following would be simpler: 
```
$ENV{HARNESS_JOBS} = $ENV{HARNESS_JOBS} // "4";
```

Reply:
or even simpler: ``ENV{HARNESS_JOBS} //= "4";

Reply:
Just remove this whole BEGIN section.  I thought it was gone already, must have given it a blind eye...

Reply:
I also thought I got rid of this! Apologies!